app/javascript/controllers/video_player_controller.js

```diff
@@ -1,6 +1,7 @@
 import { Controller } from '@hotwired/stimulus'
 import Vlitejs from 'vlitejs'
 import VlitejsYoutube from 'vlitejs/providers/youtube.js'
+import youtubeSvg from '../../assets/images/icons/fontawesome/youtube-brands-solid.svg?raw'
```

Comment:
I wonder if importing assets like that will work in production when the asset is digested? Never done it with Vite/Propshaft in the past. If needed I can try to deploy this branch on our staging environement

Reply:
That should be fine, since the JavaScript is built (and the SVG inlined) at compile-time.